### DISCUSSION TITLE
grafana-mimir helm: http://mimir-nginx.mimir-test.svc:80/prometheus 404

### DISCUSSION THREAD

**Discussion by nguyenhuukhoi** [grafana/mimir#5010] [Help and support]
Hello Grafana.
I was successful to deploy the system with k8s-helm. However when I curl http://mimir-nginx.mimir-test.svc:80/prometheus, it returned 404 not found. 
Any advices for me. thank you much,

**Comment by dimitarvdimitrov** [comment]
hi, there is no endpoint to serve on that path. You can see the list of endpoints in [this docs page](https://grafana.com/docs/mimir/latest/references/http-api/) 

You can try with `curl http://mimir-nginx.mimir-test.svc/prometheus/api/v1/status/buildinfo` to check whether mimir is running.

**Reply by nguyenhuukhoi** [reply]
Hello. It is running
curl http://mimir-nginx.mimir-test.svc/prometheus/api/v1/status/buildinfo
{"status":"success","data":{"application":"Grafana Mimir","version":"2.8.0","revision":"f917e08","branch":"HEAD","goVersion":"go1.20.4","features":{"ruler_config_api":"true","alertmanager_config_api":"true","query_sharding":"true","federated_rules":"false"}}}

**Comment by nguyenhuukhoi** [comment]
mimir running but I dont know why I curl [http://mimir-nginx.mimir-test.svc:80/prometheus](http://mimir-nginx.mimir-test.svc/prometheus), it returned 404 not found. any advices for me, ty
